torch/_tensor.py

```diff
@@ -1026,6 +1027,61 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             else:
                 return _convert(ret, cls)
 
+    def __dlpack__(self, stream=None):
+        """
+        Creates a dlpack capsule of the current tensor to
+        be exported to other libraries.
+
+        This function will be called from the `from_dlpack` method
```

Comment:
I think we can get rid of this sentence?

Reply:
I think is better to leave it as this method is not intended to be called by users, I extended it to note that `from_dlpack` passes the current stream

Reply:
OK